CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -952,9 +973,13 @@ local function CreateLeaderstatsMenuItem()
 		PlayerlistModule.TopbarEnabledChanged(enabled and not InputService.VREnabled) --We don't show the playerlist at all in VR
 	end)
 
+	local Utility = require(GuiRoot:WaitForChild("Modules"):WaitForChild("Settings"):WaitForChild("Utility"))
```

Comment:
I would, but I don't want the topbar creation to wait on it and I don't trust it to be preloaded. Maybe that's not actually a concern, but I seem to remember having an issue with it before...